bindata/network/network-metrics/000-ns.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    name: openshift-network-metrics
+    openshift.io/cluster-monitoring: "true"
+  name: openshift-network-metrics
```

Comment:
Since this is only needed because of multus, it seems like it should go in the openshift-multus namespace?

Reply:
will change.